src/chainstate/stacks/boot/pox-mainnet.clar

```diff
@@ -0,0 +1,24 @@
+;; PoX mainnet constants
+;; Min/max number of reward cycles uSTX can be locked for
+(define-constant MIN-POX-REWARD-CYCLES u1)
+(define-constant MAX-POX-REWARD-CYCLES u12)
+
+;; Default length of the PoX registration window, in burnchain blocks.
+(define-constant REGISTRATION-WINDOW-LENGTH u250)
+
+;; Default length of the PoX reward cycle, in burnchain blocks.
+(define-constant REWARD-CYCLE-LENGTH u1000)
+
+;; Valid values for burnchain address versions.
+;; TODO: These correspond to address hash modes in Stacks 2.0,
+;;    but they should just be Bitcoin version bytes: we don't
+;;    need to constrain PoX recipient addresses the way that
+;;    we constrain other kinds of Bitcoin addresses
+(define-constant ADDRESS-VERSION-P2PKH 0x00)
```

Comment:
Why are these constants duplicated in both files?

Reply:
They can be moved back into `pox.clar`, I suppose, but if we switch to using the Bitcoin version addresses, they'll need to change on mainnet/testnet.

Reply:
This is true.  While we're on that subject, if we're switching to using the bare Bitcoin address (instead of the abstracted address hash mode constants), then maybe we shouldn't validate the address version byte at all?  i.e. Bitcoin may add more address versions in the future, and we'd want to be forward-compatible with them.

Reply:
Yeah -- I think that's a good idea, but we can handle it in a separate PR. I'll open an issue for it.